packages/utils/src/rrule.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
+ */
+import { Frequency, RRule } from 'rrule';
```

Comment:
This is nice how the rrule package is only imported in one place and then we use it through the exported consts in this file. It should make maintenance easier in the future.